.github/pull_request_template.md

```diff
@@ -2,10 +2,34 @@
 
 Describe in plain language the motivation (bug, feature, etc.) behind the change in this PR and how the included commits address it.
 
-Fixes: #NNN, #NNN, ...
```

Comment:
Conventions like `Fixes:` lines are widely used and github has auto issue->pr linking based on that (https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).

Let's keep our `Fixes:` lines and have the release note automation consume them, rather than moving this down into the release notes section.

Reply:
sorry, i misunderstood your previous comment. i'll put the `Fixes:` line back.

would it also be desirable to keep the `### Bug fixes` section to allow the PR author to craft the exact changelog line or should this section be deleted and the automation script just use the issue title from `Fixes:` issues?

Reply:
I think any hand tweaking of issue descriptions should happen in the issue (or in then release notes after the tool generates the first draft) rather than the fixing PR -- let's have the tool just pull through the issue titles.

Thinking about this, there will be quite a few PRs that fix things like flaky tests which have `Fixes` lines -- that's going to need some review and/or manual review while producing notes, because there's not necessarily anything in the issue to indicate whether "failure in test foobar" was a test issue or a real bug that the test showed up.  Still, I think the issue is the right place to track that rather than the PR, even if we need to get better at describing issues clearly enough for the notes.

Reply:
got it. sound like i should delete the `### Bug fixes` section.